docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+version: '3'
+
+services:
+    scope:
+        restart: always
+        image: bujesse/scope:1.0
```

Comment:
This should be a build

Reply:
Agreed, I'll make the update

Reply:
I added a suggestion to this and also added one for the README